recipes/gatk/build.sh

```diff
@@ -21,5 +21,5 @@ mv $JAR $PACKAGE_HOME/
 
 chmod +x $PACKAGE_HOME/*.{py,jar}
 
-ln -s $PACKAGE_HOME/gatk.py $PREFIX/bin/gatk
-ln -s $PACKAGE_HOME/gatk.py $PREFIX/bin/GenomeAnalysisTK
+ln -s $PACKAGE_HOME/gatk.py $PREFIX/bin/gatk3
+ln -s $PACKAGE_HOME/gatk.py $PREFIX/bin/GenomeAnalysisTK3
```

Comment:
```suggestion
ln -s $PACKAGE_HOME/gatk.py $PREFIX/bin/GenomeAnalysisTK
```
`gatk4` does not have this and previous builds of `gatk` had it as `GenomeAnalysisTK` (i.e., we probably should keep the old name for backward compatibility): https://github.com/bioconda/bioconda-recipes/pull/7138/files#diff-9ade2138ed7598e59317b271854b6193ad573f986cd87e4565ec710795a47292L24